core/src/main/codegen/templates/Parser.jj

```diff
@@ -6767,7 +6773,8 @@ SqlCall JsonObjectFunctionCall() :
 }
 {
     <JSON_OBJECT> { span = span(); }
-    <LPAREN> [
+    <LPAREN>
```

Comment:
why this change?

Reply:
Stray edit from an earlier version of the code. I'll revert.